scripts/compose.py

```diff
@@ -265,21 +265,15 @@ class StackService(object):
 
     def image_download_url(self):
         # Elastic releases are public
-        if self.release:
+        if not self.bc:
```

Comment:
If you want to get this merged soon we could add a version limit here too, eg `self.at_least_version('6.5')`

Reply:
6.3.3 and 6.4 snapshots are now also available in the registry. I guess that should be fine? For 6.2 we use `--release` anyway, right?

Reply:
That's great news.  Agreed this should be fine as-is.